.pylintrc

```diff
@@ -16,7 +16,7 @@ ignore=CVS
 # regex matches against base names, not paths.
 ignore-patterns=.*.txt
 
-ignore-paths=obsolete/*
+ignore-paths=docs/tutorials/*
```

Comment:
@yguclu any idea why Codacy is ignoring this change?

Reply:
Not really. Codacy does not process the whole code, so it might be using the previous settings